snapshot/lib/snapshot/runner.rb

```diff
@@ -153,8 +163,7 @@ def launch(language, locale, device_type, launch_arguments)
       add_media(device_type, :video, Snapshot.config[:add_videos]) if Snapshot.config[:add_videos]
 
       open_simulator_for_device(device_type)
-
-      command = TestCommandGenerator.generate(device_type: device_type)
+      test_command = TestCommandGenerator.generate(device_type: device_type, build_type: "test-without-building")
```

Comment:
it is prepared to be not feature flagged, and ignored/forced-old-values in test_command_generator.rb